benchmarks/benchmarks/analysis/distances.py

```diff
@@ -19,112 +20,147 @@ class DistancesBench(object):
     # TODO: eventually we should include box / pbc
     # unit cell information in the benchmarks
     params = (10, 100, 1000, 10000)
-    param_names = ['num_atoms']
+    param_names = ["num_atoms"]
 
     def setup(self, num_atoms):
         np.random.seed(17809)
-        self.coords_1 = np.random.random_sample((num_atoms, 3)).astype(np.float32)
+        self.coords_1 = np.random.random_sample((num_atoms, 3)).astype(
+            np.float32
+        )
         np.random.seed(9008716)
-        self.coords_2 = np.random.random_sample((num_atoms, 3)).astype(np.float32)
-        self.allocated_array_2D = np.empty((num_atoms, num_atoms),
-                                            dtype=np.float64)
-        self.array_shape_1D = int(num_atoms * (num_atoms - 1) / 2.)
-        self.allocated_array_1D = np.empty(self.array_shape_1D,
-                                           dtype=np.float64)
-        self.u = MDAnalysis.Universe(GRO)
+        self.coords_2 = np.random.random_sample((num_atoms, 3)).astype(
+            np.float32
+        )
+        np.random.seed(15809)
+        self.coords_3 = np.random.random_sample((num_atoms, 3)).astype(
+            np.float32
+        )
+        np.random.seed(871600)
+        self.coords_4 = np.random.random_sample((num_atoms, 3)).astype(
+            np.float32
+        )
+
+        self.allocated_array_2D = np.empty(
+            (num_atoms, num_atoms), dtype=np.float64
+        )
+        self.array_shape_1D = int(num_atoms * (num_atoms - 1) / 2.0)
+        self.allocated_array_1D = np.empty(
+            self.array_shape_1D, dtype=np.float64
+        )
+        self.u = mda.Universe(GRO)
         self.ag1 = self.u.atoms[:num_atoms]
-        self.ag2 = self.u.atoms[num_atoms: 2 * num_atoms]
+        self.ag2 = self.u.atoms[num_atoms : 2 * num_atoms]
```

Comment:
```suggestion
        self.ag2 = self.u.atoms[num_atoms: 2 * num_atoms]
```

Reply:
@IAlibay Fixed now, please review it again.